src/change_stream.ts

```diff
@@ -335,6 +335,13 @@ export class ChangeStream extends EventEmitter {
     }
     return this.cursor.stream(options);
   }
+
+  tryNext(): Promise<Document | null> {
+    if (!this.cursor) {
```

Comment:
this needs to be implemented like `next` and `hasNext`:
```
return maybePromise(callback, cb => {
  getCursor(this, (err, cursor) => {
    if (err || !cursor) return cb(err); // failed to resume, raise an error
    cursor.tryNext(cb);
  });
});
```

Reply:
Also, if you are going to make this public now, please document the method, Maybe there's some configuration in tslint/eslint/tsc/api-extractor (am I missing anything? :smile:) that can require documentation for public API?

Reply:
I think [completed-docs](https://palantir.github.io/tslint/rules/completed-docs/) should do it, I'll file a ticket to add that rule.